package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-native-square-reader-sdk",
-  "version": "1.2.0",
```

Comment:
Should we update this to 1.3.0?

Reply:
It looks like @hukid is holding off until they add the API features to the plugin.

Reply:
correct, this plugin is still compatible with 1.2.1. And the plugin version is not necessarily to be aligned with Reader SDK version.